backend/tests/engine/test_epoch_settings.py

```diff
@@ -18,70 +22,103 @@
 from app.engine.user.effective_deposit.weighted_average.weights.timebased.default import (
     DefaultTimebasedWeights,
 )
+from app.engine.octant_rewards.ppf.calculator import PPFCalculatorPercent
 from app.engine.user.effective_deposit.weighted_average.weights.timebased.without_unlocks import (
     TimebasedWithoutUnlocksWeights,
 )
+from app.engine.octant_rewards import CommunityFundPercent, OctantRewardsDefaultValues
+from app.engine.octant_rewards.community_fund.not_supported import (
+    NotSupportedCFCalculator,
+)
+from app.engine.octant_rewards.ppf.not_supported import NotSupportedPPFCalculator
+from app.engine.user.budget.with_ppf import UserBudgetWithPPF
 
 
 def test_default_epoch_settings():
     register_epoch_settings()
     settings = get_epoch_settings(-1)
     check_settings(
-        settings,
+        settings=settings,
         total_and_all_individual_rewards=DefaultTotalAndAllIndividualRewards(),
         timebased_weights=TimebasedWithoutUnlocksWeights(),
         operational_cost=OpCostPercent(Decimal("0.25")),
+        ppf=PPFCalculatorPercent(OctantRewardsDefaultValues.PPF),
+        community_fund=CommunityFundPercent(OctantRewardsDefaultValues.COMMUNITY_FUND),
+        user_budget=DefaultUserBudget(),
```

Comment:
default settings for upcoming epochs should be the same as settings for epoch 3.